radio/bluetooth/bluetooth.go

```diff
@@ -76,20 +79,68 @@ func Disconnect(client ble.Client) error {
 	return nil
 }
 
-func Scan(id string, timeout time.Duration) (map[string]bool, error) {
+func WriteCharacteristic(client ble.Client, characteristic *ble.Characteristic, value []byte, noRsp bool) error {
+	err := make(chan error)
+	go func() {
+		err <- client.WriteCharacteristic(characteristic, value, noRsp)
+	}()
+
+	select {
```

Comment:
I don't think its possible to do this which is a bit annoying. It would be nice to get rid of the 4 extra lines that are needed